### PR TITLE
Add tensorboard support for local mninst example

### PR DESCRIPTION
**Pull request by eqqe**
This PR updates the mninst example to add support for local storage.
An example of usage could be:
- train your model and store the checkpoints and logs in a local storage
- deploy a tensorboard that fetches logs from the previous local storage

Local in this pull request means that the model is trained and saved on a custom PVC. Thus the results and logs of training should be accessible from Tensorboard by mounting this PVC, following  GCS and S3 examples.

Files changed:
- mnist/README.md: New instructions on how to use the logs found in the local storage for monitoring, following the same scheme as the "Training your model" step.
- mnist/monitoring/local/deployment_patch.yaml: Mount the local-storage PVC created in the training step to be accessible by Tensorboard.
- mnist/monitoring/local/kustomization.yaml: Define pvc name and mount path to be accessible in tensorboard
- mnist/monitoring/local/params.yaml: Define claimName and mountPath.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/examples/616)

### CODE REVIEW

**Comment by eqqe** [comment]
@googlebot I signed it!

**Comment by jlewi** [comment]
@jinchihe do you want to review this?

@eqqe Would you mind expanding the PR description to include a list of the changes? Can you explain in the PR description what local means in this context?
 
/assign @jinchihe

File: mnist/README.md [hunk 1]
```diff
@@ -469,6 +469,21 @@ There are various ways to monitor workflow/training job. In addition to using `k
 
 ### Tensorboard
 
+#### Local storage
```
Comment (jinchihe) [inline mnist/README.md]: @eqqe Cloud you please update the Contents of Readme file to add the section? Thanks.

**Comment by jinchihe** [comment]
@eqqe Thanks for the contribution! Others are great.

**Comment by jinchihe** [comment]
Nice work!
/lgtm
/approve

**Pull request merged**